pkg/systemd/overview-cards/healthCard.jsx

```diff
@@ -31,22 +31,20 @@ import "./healthCard.scss";
 
 const _ = cockpit.gettext;
 
-export class HealthCard extends React.Component {
-    render() {
-        return (
-            <Card className="system-health">
-                <CardTitle>{_("Health")}</CardTitle>
-                <CardBody>
-                    <ul className="system-health-events">
-                        <PageStatusNotifications />
-                        <InsightsStatus />
-                        <CryptoPolicyStatus />
-                        <ShutDownStatus />
-                        <LastLogin />
-                    </ul>
-                </CardBody>
-                <CardFooter />
-            </Card>
-        );
-    }
-}
+export const HealthCard = () => {
+    return (
+        <Card className="system-health">
```

Comment:
Style issue: For such a "simple mapping" component, return the element directly: `HealthCard = () => <Card>`, or if you want to keep the (admittedly nicer) indendation on the next line, use `HealthCard = () => (` and save both the extra `{ return }` and the extra indendation.

Reply:
Excellent point!